task/clamav-scan/0.1/clamav-scan.yaml

```diff
@@ -141,6 +144,38 @@ spec:
           name: dbfolder
         - mountPath: /work
           name: work
+    - name: upload
+      image: quay.io/konflux-ci/oras:latest@sha256:55cf304549225435958798c3be0377faf0a3ff89f3fbe03e45c3bb656ecaf02f
+      env:
+        - name: IMAGE_URL
+          value: $(params.image-url)
+        - name: IMAGE_DIGEST
+          value: $(params.image-digest)
+      workingDir: /work
+      script: |
+        #!/usr/bin/env bash
+
+        IFS=':' read -ra REPO <<< "$IMAGE_URL"
+        REPO="${REPO[0]}"
+        DIGEST=$(echo "${IMAGE_DIGEST}" | tr : -)
+        SUFFIX=clamav
+        CLAMAV_RESULT_URI="${REPO}:${DIGEST}.${SUFFIX}"
+
+        cd logs
+
+        for UPLOAD_FILE in $(find . -name "clamscan-result*.log"); do
+          MEDIA_TYPE=text/vnd.clamav
+          args+=("${UPLOAD_FILE}:${MEDIA_TYPE}")
+        done
+        for UPLOAD_FILE in $(find . -name "clamscan-ec-test*.json"); do
+          MEDIA_TYPE=application/vnd.konflux.test_output+json
+          args+=("${UPLOAD_FILE}:${MEDIA_TYPE}")
+        done
+        retry oras push --no-tty --registry-config "$HOME/.docker/config.json" "${CLAMAV_RESULT_URI}" "${args[@]}"
```

Comment:
shouldn't we use `oras attach` for creating an artifacts graph?

Reply:
Done.